slam/surface_profiling.py

```diff
@@ -0,0 +1,354 @@
+
+import numpy as np
+import slam.geodesics
+import trimesh
+
+
+def cortical_surface_profiling(mesh, theta, r_step, m, save_path):
+    """
+
```

Comment:
please provide description

Reply:
Important also to have correct names for the variables. For instance, mesh, r_step, save_path are relatively straightforward but theta or m far less.